docker/dockerfiles/test-docker-compose.yaml

```
@@ -1,14 +1,14 @@
 version: '3'
 services:
   relay_alice:
-    image: horizenlabs/zkverify:0.7.0-0.9.0-relay
+    image: horizenlabs/zkverify:local-relay
```

This is on my branch already
```suggestion
    image: horizenlabs/zkverify:latest-relay
```

---

That's a little bit opinionated: `horizenlabs/zkverify:latest-relay` is compiled without fast runtime feature and an epoch is 1 hour long. That means you need 1 h to have the parachain up and running.

Maybe is better to document it. Otherwise we can think to deploy also a `horizenlabs/zkverify:latest-fastruntime` docker image.

---

Yes, whatever works the best. If we go with `local-relay` we need to add steps under README on how to build it as well. I am opened to suggestions :wink: